Tests/LockableSpec.swift

```diff
@@ -49,6 +49,50 @@ class LockableSpec: QuickSpec {
 
                 expect(lockableObject.resource.get()).toEventually(equal([1, 2, 3, 4]))
             }
+
+            it("will make other threads wait to execute lock() call") {
+                let resource = lockableObject.resource
+                resource.lock()
+                DispatchQueue.global().async {
+                    resource.lock()
+                    expect(resource.get()).to(equal([1]))
+                }
+                resource.set(value: [1])
+                expect(resource.isLocked).to(beTrue())
+                resource.unlock()
```

Comment:
This test is a little difficult to follow. Maybe some comments would help.

Reply:
Done. `--verbose` added to the test 😄